app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -1,27 +1,46 @@
-import React, { useState } from "react";
+import React, { useState, useContext } from "react";
 
 import { Formik, Form } from "formik";
-import { Button, Pane } from "neetoui/v2";
-import { Input, Textarea } from "neetoui/v2/formik";
+import moment from "moment";
+import { Toastr, Button, Pane } from "neetoui/v2";
+import { Input, Textarea, Select } from "neetoui/v2/formik";
 
-import notesApi from "apis/notes";
 import formValidationSchemas from "constants/formValidationSchemas";
+import { NoteContext } from "contexts/note";
 
-export default function NoteForm({ onClose, refetch, note, isEdit }) {
+import {
+  FORM_TAGS_DROPDOWN,
+  FORM_CONTACTS_DROPDOWN,
+  SAMPLE_NOTES,
+} from "../constants";
+
+export default function NoteForm({ note, isEdit }) {
   const [submitted, setSubmitted] = useState(false);
+  const { setShowNewNotePane } = useContext(NoteContext);
+
   const handleSubmit = async values => {
-    try {
-      setSubmitted(true);
-      if (isEdit) {
-        await notesApi.update(note.id, values);
-      } else {
-        await notesApi.create(values);
-      }
-      refetch();
-      onClose();
-    } catch (err) {
-      logger.error(err);
-    }
+    const day = new Date().toLocaleString("en-US", {
+      weekday: "long",
+    });
+
+    const time = new Date().toLocaleString("en-US", {
+      hour: "numeric",
+      minute: "numeric",
+      hour12: true,
+    });
+
+    const createdAt = moment().fromNow();
```

Comment:
Please use `dayjs` for all date-related calculations. 
We don't recommend using `moment` in projects anymore. 
Ref - https://github.com/bigbinary/neeto-engineering/blob/master/Front-End-Guidelines/react-guidelines/dayjs-over-momentjs.md